client/.eslintrc

```diff
@@ -4,5 +4,9 @@
     "browser": true,
     "node": true,
     "jest": true
+  },
+  "rules": {
+    "no-duplicate-imports": 0,
```

Comment:
I'd like to have that too but no comments in json (and I feel it's too minor for readme.md)

it's because no-duplicate-imports can't deal with import type and import/no-duplicates can

Reply:
You can :) https://github.com/mozilla/addons-frontend/issues/942